flowers.json

```diff
@@ -282,7 +282,7 @@
       ],
       "blooming_period": "--05-01/--10-31",
       "description": {
-        "summary": "Trifolium repens, or white clover, is native from Europe to western Asia and into north Africa but has been introduced as a forage species for livestock throughout the world. Although it isn’t native to the area, it is often used for foraging by bees. White clover is often used by honey bees for nectar, which then makes clover honey. Some people are replacing their grass lawns with clover because it’s easier and cheaper to take care of. White clover has green stem and leaves with white flowering heads and flowers from May to October.",
+        "summary": "White clover is native from Europe to western Asia and into north Africa but has been introduced as a forage species for livestock throughout the world. Although it isn’t native to the area, it is often used for foraging by bees. White clover is often used by honey bees for nectar, which then makes clover honey. Some people are replacing their grass lawns with clover because it’s easier and cheaper to take care of. White clover has green stem and leaves with white flowering heads and flowers from May to October.",
```

Comment:
Another instance of "Typesetter's apostrophe", see the comment further down on rationalizing these.